newdle/templates/base_email.html

```diff
@@ -31,7 +31,7 @@
     <tr>
       <td style="text-align: center; color: #aaa; font-size: 11px; padding-bottom: 10px;">
         <p style="padding: 0 50px; margin-top: 10px;">
-          This email was sent by <a href="{{ url_for('index', _external=true) }}">newdle</a>.
+          {% trans %}This email was sent by{% endtrans %} <a href="{{ url_for('index', _external=true) }}">newdle</a>.
```

Comment:
Don't have partial sentences in translatable strings. The link must be part of it.  You can do something like this:

```jinja
{% set newdle_link -%}
    <a href="{{ url_for('index', _external=true) }}">newdle</a>
{%- endset %}
{% trans %}This email was sent by {{ newdle_link }}.{% endtrans %}
```